Documentation/configuration/metrics.rst

```diff
@@ -151,7 +151,7 @@ contact the Kubernetes API server for these lists and must have permissions to
 do so.
 
 An example of a Prometheus configuration can be found alongside the reference
-Cilium Kubernetes DaemonSet spec in :git-tree:`examples/kubernetes/addons/prometheus/prometheus.yaml`
+Cilium Kubernetes DaemonSet spec in :git-tree:`examples/kubernetes/addons/prometheus/04-prometheus.yaml`
```

Comment:
this link is also dead, should be `templates/...`

Reply:
:man_facepalming:  sorry, should be ok now @aanm